src/test/java/fr/esgi/al5_2/Tayarim/ProprietaireServiceTest.java

```diff
@@ -26,262 +25,281 @@
+        .isPasswordUpdated(Boolean.TRUE)
+        .build();
+    proprietaire.setLogements(null);
+    proprietaire.setId(1L);
+    Proprietaire proprietaire2 = Proprietaire.builder()
+        .nom("Ferreira")
+        .prenom("Mathieu")
+        .email("test@gmail.com")
+        .numTel("0612345678")
+        .motDePasse("password")
+        .dateInscription(localDateTime)
+        .isPasswordUpdated(Boolean.TRUE)
+        .build();
+    proprietaire2.setLogements(null);
+    proprietaire2.setId(2L);
+    when(proprietaireRepository.findAll()).thenReturn(List.of(proprietaire, proprietaire2));
+
+    List<ProprietaireDto> proprietaireDtos = proprietaireService.getProprietaire(false);
+
+    Assertions.assertThat(proprietaireDtos).isNotNull();
+    Assertions.assertThat(proprietaireDtos).hasSizeGreaterThan(1);
+
+  }
+
+  @Test
+  public void ProprietaireService_GetProprietaireById_ReturnOneProprietaireDto() {
+    LocalDateTime localDateTime = LocalDateTime.now();
+
+    Proprietaire proprietaire = Proprietaire.builder()
+        .nom("Ferreira")
+        .prenom("Mathieu")
+        .email("test@gmail.com")
+        .numTel("0612345678")
+        .motDePasse("password")
+        .dateInscription(localDateTime)
+        .isPasswordUpdated(Boolean.TRUE)
+        .build();
+    proprietaire.setLogements(null);
+    proprietaire.setId(1L);
+    when(proprietaireRepository.findById(proprietaire.getId())).thenReturn(
+        Optional.of(proprietaire));
+
+    ProprietaireDto proprietaireDTO = proprietaireService.getProprietaireById(proprietaire.getId(),
+        false);
+
+    Assertions.assertThat(proprietaireDTO).isNotNull();
+    Assertions.assertThat(proprietaireDTO.getId()).isEqualTo(proprietaire.getId());
+    Assertions.assertThat(proprietaireDTO.getNom()).isEqualTo(proprietaire.getNom());
+    Assertions.assertThat(proprietaireDTO.getPrenom()).isEqualTo(proprietaire.getPrenom());
+    Assertions.assertThat(proprietaireDTO.getEmail()).isEqualTo(proprietaire.getEmail());
+    Assertions.assertThat(proprietaireDTO.getNumTel()).isEqualTo(proprietaire.getNumTel());
+    Assertions.assertThat(proprietaireDTO.getDateInscription())
+        .isEqualTo(proprietaire.getDateInscription());
+
+  }
+
+  @Test
+  public void ProprietaireService_GetProprietaireByEmail_ReturnOneProprietaireDto() {
+    LocalDateTime localDateTime = LocalDateTime.now();
+
+    Proprietaire proprietaire = Proprietaire.builder()
+        .nom("Ferreira")
+        .prenom("Mathieu")
+        .email("test@gmail.com")
+        .numTel("0612345678")
+        .motDePasse("password")
+        .dateInscription(localDateTime)
+        .isPasswordUpdated(Boolean.TRUE)
+        .build();
+    proprietaire.setLogements(null);
+    proprietaire.setId(1L);
+    when(proprietaireRepository.findFirstByEmail(proprietaire.getEmail())).thenReturn(
+        Optional.of(proprietaire));
+
+    ProprietaireDto proprietaireDTO = proprietaireService.getProprietaireByEmail(
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.AbbreviationAsWordInNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Abbreviation in name 'proprietaireDTO' must contain no more than '1' consecutive capital letters.